src/services/jsDoc.ts

```diff
@@ -68,6 +68,33 @@ namespace ts.JsDoc {
         return documentationComment;
     }
 
+    export function getJsDocTagsFromDeclarations(declarations: Declaration[]) {
+        // Only collect doc comments from duplicate declarations once.
+        const tags: JSDocTagInfo[] = [];
+        forEachUnique(declarations, declaration => {
+            const jsDocs = getJSDocs(declaration);
+            if (!jsDocs) {
+                return;
+            }
+            const jsDocTags: JSDocTag[] = [];
+            jsDocs.forEach(doc => {
+                const tagsForDoc = (doc as JSDoc).tags;
+                if (tagsForDoc) {
+                    jsDocTags.push(...tagsForDoc.filter(tag => tag.kind === SyntaxKind.JSDocTag));
+                }
+            });
+            if (!jsDocTags) {
+                return;
+            }
+            for (const tag of jsDocTags) {
+                if (tag) {

```

Comment:
can it ever be `null` or `undefined`? I think in this case line `83` should fail